conanfile.py

```diff
@@ -266,34 +280,41 @@ def _openssl_path(self):
             pck = "OpenSSL"
         return self.conanfile.deps_cpp_info[pck].rootpath
 
-    def _build(self, dest_dir):
-        from conan.tools.gnu import AutotoolsToolchain, Autotools
-
+    def generate(self, dest_dir):
+        from conan.tools.gnu import AutotoolsToolchain, AutotoolsDeps
         tc = AutotoolsToolchain(self.conanfile, prefix=dest_dir)
         tc.configure_args.extend(["--enable-shared", f"--with-openssl={self._openssl_path}"])
-        tc.make_args.append("-j8")
```

Comment:
AFAIK the new generators still do single-process builds by default. Can you add some parallelization back? The hardcoded `-j8` was not great, but I think we can do it better with `build.build_jobs()` now: https://docs.conan.io/1/reference/conanfile/tools/build.html#conan-tools-build-build-jobs

Reply:
Oh interesting. I saw this
```python
    njobs = conanfile.conf.get("tools.build:jobs",
                               default=_cpu_count(),
                               check_type=int)
```
and automatically assumed it gave the right cpu_count.

Will bring the parameter back.

Reply:
Maybe it does. Best to check locally. My knowledge may be outdated (from the old version of the Conan autotools helpers).

Reply:
```python
def _cpu_count():
    try:
        try:
            # This is necessary to deduce docker cpu_count
            cfs_quota_us = cfs_period_us = 0
            # cgroup2
            if os.path.exists("/sys/fs/cgroup/cgroup.controllers"):
                cpu_max = load("/sys/fs/cgroup/cpu.max").split()
                if cpu_max[0] != "max":
                    if len(cpu_max) == 1:
                        cfs_quota_us, cfs_period_us = int(cpu_max[0]), 100_000
                    else:
                        cfs_quota_us, cfs_period_us = map(int, cpu_max)
            else:  # cgroup1
                cfs_quota_us = int(load("/sys/fs/cgroup/cpu/cpu.cfs_quota_us"))
                cfs_period_us = int(load("/sys/fs/cgroup/cpu/cpu.cfs_period_us"))
            if cfs_quota_us > 0 and cfs_period_us > 0:
                return int(math.ceil(cfs_quota_us / cfs_period_us))
        except (EnvironmentError, TypeError):
            pass
        return multiprocessing.cpu_count()
    except NotImplementedError:
        # print("multiprocessing.cpu_count() not implemented. Defaulting to 1 cpu")
        return 1  # Safe guess
```
Implementation looks like this. So I guess `multiprocessing` should be good enough at this at this point?

Reply:
At least in my system it pass -j12. Where was this supposed to fail?

Reply:
If it works fine, then please disregard my comment. I was basing this on the old tooling.